tests/src/FunctionalJavascript/ListPageRssControllerTest.php

```diff
@@ -48,39 +49,56 @@ public function testListPageRssPage() {
     $page->pressButton('Save');
 
     // Create some test nodes to index and search in.
-    $date = new DrupalDateTime('20-10-2020');
+    $earlier_date = new DrupalDateTime('20-10-2020');
+    $later_date = new DrupalDateTime('20-10-2021');
     $values = [
       'title' => 'that yellow fruit',
       'type' => 'content_type_one',
       'body' => 'this is a banana',
       'status' => NodeInterface::PUBLISHED,
       'field_select_one' => 'test1',
-      'created' => $date->getTimestamp(),
+      'created' => $earlier_date->getTimestamp(),
+      'changed' => $later_date->getTimestamp(),
     ];
     $node = Node::create($values);
     $node->save();
+
     $values = [
       'title' => 'that red fruit',
       'type' => 'content_type_one',
       'body' => 'this is a cherry',
       'field_select_one' => 'test2',
       'status' => NodeInterface::PUBLISHED,
-      'created' => $date->getTimestamp(),
+      'created' => $later_date->getTimestamp(),
+      'changed' => $earlier_date->getTimestamp(),
     ];
     $node = Node::create($values);
     $node->save();
     /** @var \Drupal\search_api\Entity\Index $index */
     $index = Index::load('node');
-    // Index the nodes.
     $index->indexItems();
 
     $node = $this->drupalGetNodeByTitle('List page test');
     $this->drupalLogout();
 
-    // Assert contents of channel elements.
+    // Assert the default sorting order of the list is
+    // by creation date, descending.
+    $this->drupalGet(Url::fromRoute('entity.node.canonical', ['node' => $node->id()]));
```

Comment:
Not sure you need all this. You can simply just deduce the order below when you do things like `$first_item = $items->eq(0);`

Reply:
I'm first checking that the normal list is ordered by creation date and then through the other tests we assert that the rss list is ordered by the changed.

Reply:
@imanoleguskiza I know what you are doing. But below you are doing `$first_item = $items->eq(0);` which means you expect the first one (0) to be something. So with that you also assert the order cause you only have 2 items. You dont need a separate thing.